wicket/src/screens/common.rs

```diff
@@ -134,19 +139,53 @@ impl CommonScreenState {
         }
     }
 
-    pub fn draw_screen_selection_buttons(&self, f: &mut Frame) {
-        // Draw the RackSreenButton
-        let border_style =
-            if self.hovered == Some(self.rack_screen_button_state.id()) {
-                style::button_hovered()
-            } else {
-                style::button()
+    pub fn draw_screen_navigation_instructions(&self, f: &mut Frame) {
+        let prev = self.prev_screen_label();
+        let next = self.next_screen_label();
+        if let Some(p) = prev {
+            let width = p.width() as u16;
+            let prev =
+                Paragraph::new(p).style(Style::default().fg(OX_OFF_WHITE));
+            let rect = Rect {
+                x: f.size().width - width - next.width() as u16 - 3,
```

Comment:
Constants like ` - 3` make me a little uneasy, but I'm not going to quibble too much. Does this have a meaningful name, or is it relative to something else?

Reply:
Yeah, it's an offset for spacing, I'll name it. Good call!